dataretrieval/nwis.py

```diff
@@ -429,17 +430,28 @@ def get_gwlevels(
     .. doctest::
 
         >>> # Get groundwater levels for site 434400121275801
-        >>> df, md = dataretrieval.nwis.get_gwlevels(sites='434400121275801')
+        >>> df, md = dataretrieval.nwis.get_gwlevels(site_no='434400121275801')
```

Comment:
```suggestion
        >>> df, md = dataretrieval.nwis.get_gwlevels(sites='434400121275801')
```

Reply:
What do you think about this last suggestion? It isn't a big deal, but I think "sites" is more consistent with the rest of the nwis module.